README.md

```diff
@@ -5,9 +5,8 @@ This is a DDEV addon for doing Drupal core development.
 ```
 git clone https://git.drupalcode.org/project/drupal.git drupal
 cd drupal
-ddev config --project-type=drupal10
+ddev config --project-type=drupal --omit-containers=db 
```

Comment:
```suggestion
ddev config --omit-containers=db --disable-settings-management
```
the only two flags necessary are to omit the database container that it isnt created on the first ddev start. and disable settings management is mandatory in the initial config otherwise ddev is adding a link to settings.ddev.php into the settings.php and so on. 

the project type is not necessary same for most of the other settings we had before. the cloned git repo is automatically recognized as drupal 11, in consequence the project is set to php83 and corepack enabled (see https://github.com/ddev/ddev/blob/18e53dcc49440f621596387b40d1b942333e1587/pkg/ddevapp/drupal.go).